app/controllers/application_controller.rb

```diff
@@ -1,2 +1,18 @@
 class ApplicationController < ActionController::Base
+  before_action :set_current_user
+
+  private
+
+  def set_current_user
+    user_id = params[:user_id]
+    @current_user = User.find_by(id: user_id)
+  end
```

Comment:
- [ ] As required for the project, kindly ensure that the `#set_current_user` method you created returns the first user from the database as now it is not the case. It should to something similar to this:
```ruby
def current_user
  User.first
end
```